### PR TITLE
v1.8.3

### PR DESCRIPTION
**Pull request by creisle**
Improvements
- prints log errors when the validation step was run after annotations (a partial rerun)
- standardization of spanning read calls
- pairing states for transcriptome vs genome using either pairing type (pairing, inferred_pairing)

Bug Fixes
- read evidence undercalled due to strand being set for unstranded sources
- explicit hashing of reads to avoid over-counting

### CODE REVIEW

File: mavis/breakpoint.py [hunk 1]
```diff
@@ -419,6 +419,50 @@ def breakpoint_sequence_homology(self, reference_genome):
...
+                else:
+                    break
+        else:
+            break2_seq = reference_genome[self.break2.chr].seq[self.break2.start - 1: self.break2.start + lutemp - 1]
+            for i in range(0, lutemp):
+                if break2_seq[i] == self.untemplated_seq[i]:
+                    break2_shift += 1
+                else:
+                    break
+        return (break2_shift, break1_shift)
```
Comment (dpaulino) [inline mavis/breakpoint.py]: Is the logic for determining `break2_shift` and `break1_shift` identical? If so, the logic should be put into a method and just called twice

File: mavis/summary/summary.py [hunk 2]
```diff
@@ -219,21 +219,21 @@ def get_pairing_state(current_protocol, current_disease_state, other_protocol, o
...
+        return PAIRING_STATE.GERMLINE if is_matched else PAIRING_STATE.SOMATIC
     elif curr == ng and other == dt:
-        return PAIRING_STATE.EXP if inferred_is_matched else PAIRING_STATE.NO_EXP
+        return PAIRING_STATE.EXP if is_matched else PAIRING_STATE.NO_EXP
     else:
-        if current_protocol == other_protocol:
-            return PAIRING_STATE.MATCH if is_matched else PAIRING_STATE.NO_MATCH
-        else:
-            return PAIRING_STATE.MATCH if inferred_is_matched else PAIRING_STATE.NO_MATCH
+        return PAIRING_STATE.MATCH if is_matched else PAIRING_STATE.NO_MATCH
```
Comment (dpaulino) [inline mavis/summary/summary.py]: looks like some of these conditions can be merged together to improve clarify

Comment (creisle) [reply mavis/summary/summary.py]: not sure why this isn't showing up in the full diff. I made these changes

Comment (dpaulino) [reply mavis/summary/summary.py]: That's weird. I see the change in that commit. But on the main pr page it doesn't show up. anyways, thanks

**Pull request merged**